.drone.yml

```diff
@@ -27,26 +27,30 @@ steps:
 - name: cargo-format
   image: rust:1.64
   commands:
-  - rustup component add rustfmt &&
-    cargo fmt --all -- --check
+  - rustup component add rustfmt && cargo fmt --all -- --check
   environment:
     CARGO_HOME: /drone/src/.cargo
   depends_on:
   - cargo-deps
 
+- name: cargo-doc-ci
+  image: rust:1.64
+  commands:
+  - cargo doc --document-private-items --workspace --all-features --no-deps
+  environment:
+    RUSTDOCFLAGS: -Dwarnings
+  depends_on:
+  - cargo-format
+
 - name: cargo-clippy-ci
   image: rust:1.64
   commands:
-  - rustup component add clippy &&
-    cargo clippy -- -D warnings &&
-    cargo clippy --features toggle -- -D warnings &&
-    cargo clippy --release -- -D warnings &&
-    cargo clippy --release --features toggle -- -D warnings
+  - rustup component add clippy && cargo clippy --workspace -- -D warnings && cargo clippy --workspace --all-features -- -D warnings && cargo clippy --workspace --release -- -D warnings && cargo clippy --workspace --release --all-features -- -D warnings
```

Comment:
drone fmt really just decided to fuck this line up huh

Reply:
Wtf, why does it even bother parsing commands, that sounds dangerous in itself

Reply:
Oh wait no I see it already has && it just flattened the list into a single value. Why even bother

Reply:
fixed